js/freewheel.js

```diff
@@ -634,6 +638,22 @@ OO.Ads.manager(function(_, $) {
     };
 
     /**
+     * This function is called by the ad manager controller when a video element comes into focus.
+     * @param videoId   the id of the video element that came into focus
+     */
+    this.videoElementInFocus = function(videoId) {
+      // The Freewheel SDK does not like when the video elements passed in via _registerDisplayForLinearAd
+      // and _registerDisplayForNonlinearAd have display set to none. This causes overlays to not
+      // be positioned and sized correctly (PBI-1307). When we get notified that the main video content
+      // is in focus, we need update the overlay position.
+      // Note: Updating overlay position uses undocumented methods and may break at any time. Also, simply
+      // notifying the SDK via _registerDisplayForNonlinearAd does not appear to be sufficient at this time
+      if (videoId === OO.VIDEO.MAIN) {
+        updateOverlayPosition();
+      }
```

Comment:
Hm - the one thing i'm feeling mixed about is the fact that now freewheel has to be aware of what videoIds are.  Also I would advise using an ad manager "event" for notifications rather than a function so that it's only called on the subscribed plugins and is called on all plugins